content/docs/2.7/concepts/authentication.md

```diff
@@ -272,37 +273,4 @@ podIdentity:
   provider: none | azure | aws-eks | aws-kiam  # Optional. Default: none
 ```
 
-#### Azure Pod Identity
```

Comment:
All of the above are also identity providers: 
- Azure Key Vault secret(s) 
- Hashicorp Vault secret(s) 
- Secret(s)
- Environment variable(s)

Can we add those in individual files as well? If it helps; I'm happy to do so in another PR.

Reply:
Done!

Reply:
@tomkerkhove, you mentioned those 4 "identity providers", whcih @cynthia-sg has added. These now show up under https://deploy-preview-867--keda.netlify.app/docs/2.7/providers/, which has as a title "Authentication Providers". Maybe the section title should be changed to just **Providers**? (If we agree to the title change, then it should be changed everywhere -- i.e., including for the corresponding nav entry titles.)

If so, then the first paragraph in that page should be changed too, maybe:

> Available authentication and identity providers for KEDA:

WDYT?

Reply:
I would not rename them to "providers" because it's not clear what the providers are for - Are they autoscaling providers or authentication providers?

That's why I would stick with "authentication providers" and also include this in the URL (be it "auth-providers", "authentication-providers", "providers/auth/xxx").

What do you think @zroubalik?

Reply:
I'm using right now `authentication-providers`, wdyt?

> > > > @tomkerkhove - This PR presents the **Authentication Providers** in the left nav as a drop down. Do you expect this list of providers to grow like for that for Scalers? If so, consider not listing the providers via a drop down.
> > > 
> > > 
> > > I don't think we'll have more than 10 or so long term but it does not have to be a dropdown perse; I would just like to have a shortcut here: ![image](https://user-images.githubusercontent.com/4345663/182849130-278995cc-c1ec-4f25-bf13-a45ad50d3cc0.png)
> > 
> > 
> > The link for the `Authentication Providers` is there in fact. I added it for mobile version but it's hidden for desktop using the class `providers-link`. I agree with @chalin about the navbar is crowded and perhaps it's necessary to change this.
> 
> Fine by me, but then it should be in this drop-down at least because it's hidden atm: ![image](https://user-images.githubusercontent.com/4345663/183361186-88211a96-5e32-462a-8c39-3373dd6a8f5a.png)

I have added the link to the dropdown and removed the previous one.

<img width="629" alt="keda-link" src="https://user-images.githubusercontent.com/1440981/184503223-322b8c4a-ead5-493f-bb33-eed879dd6b32.png">

Reply:
> I would not rename them to "providers" because it's not clear what the providers are for - Are they autoscaling providers or authentication providers?
> 
> That's why I would stick with "authentication providers" and also include this in the URL (be it "auth-providers", "authentication-providers", "providers/auth/xxx").
> 
> What do you think @zroubalik?

I'm used `authentication-providers` as you suggested @tomkerkhove.